gno.land/pkg/gnoclient/client_test.go

```diff
@@ -1079,3 +1079,116 @@ func TestAddPackageErrors(t *testing.T) {
 		})
 	}
 }
+
+// Block tests
+func TestBlockErrors(t *testing.T) {
```

Comment:
Please add unit tests that verify the functionality, not just the errors 🙏

Reply:
I'm not sure to what degree the mocks should be written - is [d428c1d](https://github.com/gnolang/gno/pull/1910/commits/d428c1da3d3b98bd93996d734883f241d798eb04) & [8951210](https://github.com/gnolang/gno/pull/1910/commits/8951210bce076b477f3d1c4bc90a9d24065c7702) enough?